src/content/advanced/cluster-management/new-cluster-api/index.md

```diff
@@ -0,0 +1,124 @@
+
+* **Lack of On-Demand Updates:** Operator bug fixes required a full release and upgrade cycle for both GS and customers, hindering timely remediation.
+
+CAPI addresses these limitations with a new approach:
+
+* **Cross-Version Compatibility:** New operator versions can manage all previously provisioned clusters, ensuring ongoing support for existing deployments.
+
+* **On-Demand Upgrades:** Cluster upgrades are triggered explicitly per cluster, preventing unplanned or unintended upgrades.
+
+* **Enhanced Testing for Operators:** Thorough testing ensures the quality and reliability of the operators.
+
+### Flexible Network Configuration
+
+CAPI introduces greater flexibility in defining network configurations, allowing for more granular control over cluster topology:
+
+* **Similar Defaults to Vintage:** The default network setup aligns with the familiar Vintage approach.
+
+* **Separate Subnets for Key Components:** The API Load Balancer (LB), Ingress LB, control plane nodes, and worker nodes can be deployed on distinct subnets, enhancing isolation and security.
+
+* **Transit Gateway Support**
+
+CAPI fully integrates with Transit Gateway, enabling seamless connectivity across multiple AWS accounts and regions. This flexibility expands the range of deployment options.
+
+### Removal of Initiator App
+
+The legacy Initiator app has been removed due to its limitations and potential security risks. Cluster configuration is now solely managed through the configurable options provided by the `cluster-aws` Helm chart. This approach ensures better testing and quality control.
+
+### Private Clusters with Limitations
+
+CAPI supports private clusters with the following limitations:
+
+* **Private Kubernetes API LB:** The Kubernetes API LB can be configured to be private, restricting external access.
+
+* **No Bastion Hosts:** Bastion hosts are no longer used for secure access to private clusters.
+
+* **HTTP/HTTPS Proxy Required:** A HTTP/HTTPS proxy is necessary for communication with the CAPA API.
+
+### Teleport for Secure Cluster Access
+
+CAPI introduces Teleport as the primary method for secure cluster access, replacing bastion hosts. Teleport offers several advantages:
+
+* **Removal of Bastion Hosts:** Bastion hosts are no longer required, reducing attack surfaces and simplifying security management.
+
+* **SSH and kubectl Audit Logs:** Session recording capabilities for SSH and kubectl commands provide comprehensive audit logs.
+
+* **SSO for SSH and Kubernetes API Access:** Single sign-on (SSO) simplifies access and enhances security.
+
+* **Private Cluster Enablement:** Teleport enables private clusters without publicly exposed endpoints.
+
+* **Reduction in VPN Usage:** GS VPN may no longer be needed, except for Grafana, Prometheus, and Happa (if deployed externally).
```

Comment:
```suggestion
### Teleport for Secure Cluster Access

Our Cluster API implementation introduces [Teleport](https://goteleport.com/) as the primary method for secure cluster access, replacing bastion hosts. The advantages of this approach are:

* **Removal of Bastion Hosts:** Bastion hosts are no longer required, reducing attack surfaces and simplifying security management.

* **SSH and kubectl Audit Logs:** Session recording capabilities for SSH and kubectl commands provide comprehensive audit logs.

* **SSO for SSH and Kubernetes API Access:** Single sign-on (SSO) simplifies access and enhances security.

* **Private Cluster Enablement:** Teleport enables private clusters without publicly exposed endpoints.

* **Reduction in VPN Usage:** Giant Swarm VPN may no longer be needed, except for Grafana, Prometheus, and Happa, in case those are deployed externally.
```